### PR TITLE
Runtime 8 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -24,6 +24,18 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
 
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Astro Runtime 8.0.0
+
+- Release date: TBD
```
Comment (jwitz) [inline astro/runtime-release-notes.md:29]:
```suggestion
- Release date: April 30, 2023
```

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -24,6 +24,18 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+- Airflow version: 2.6.0
+
+### Airflow 2.6
+
+Astro Runtime 8 is based on Airflow 2.6, which includes a number of new features and improvements with an emphasis on observability. Most notably, Airflow 2.6 includes the following changes:
+
+- [Notifiers](https://airflow.apache.org/docs/apache-airflow/latest/howto/notifications.html) are a new class that can be used to send notifications from a DAG to a third party application, such as Slack. This release includes the [SlackNotifier](https://airflow.apache.orgdocs/apache-airflow-providers-slack/latest/_api/airflow/providers/slack/notifications/slack_notifier/index.html#airflow.providers.slack.notifications.slack_notifier.SlackNotifier), with more notifiers coming in the future. 
+- A major bug related to zombie tasks has been fixed. The logic for handling stalled tasks has been moved to the scheduler, and any tasks that have been queued for more than `scheduler.task_queued_timeout` are now marked as failed. This prevents a type of zombie task where tasks are stuck in an infinite loop of being scheduled and queued.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:37]:
```suggestion
- A major bug related to zombie tasks has been fixed. The logic for handling stalled tasks has been moved to the scheduler, and any tasks that have been queued for more than `scheduler.task_queued_timeout` are now marked as failed. This prevents a type of zombie task where tasks are stuck in an infinite loop of being scheduled and queued. 
- You can now view logs for individual triggers in the Airflow UI. Trigger logs tell you when an individual task is sleeping and when it's triggered.

To learn more, see the [Apache Airflow 2.6.0 release notes](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-6-0-2023-04-30).
```

File: astro/runtime-version-lifecycle-policy.md [hunk 3]
```diff
@@ -87,6 +87,7 @@ The following table contains the exact lifecycle for each published version of A
 | [5](runtime-release-notes.md#astro-runtime-500) | 2.3                    | April 30, 2022     | October 2023            |
 | [6](runtime-release-notes.md#astro-runtime-600) | 2.4                    | September 19, 2022 | March 2024              |
 | [7](runtime-release-notes.md#astro-runtime-700) | 2.5                    | December 3, 2022   | June 2023               |
+| [8](runtime-release-notes.md#astro-runtime-800) | 2.6                    | TBD                | October 2023            |
```
Comment (jwitz) [inline astro/runtime-version-lifecycle-policy.md:90]:
```suggestion
| [8](runtime-release-notes.md#astro-runtime-800) | 2.6                    | April 30, 2023                | October 2023            |
```

**Review by manmeetkaur — APPROVED** [review]

**Review by paolaperaza — APPROVED** [review]
LGTM! There may be a few Airflow 2.6 features missing that we might want to add later, but this is good

**Pull request merged**